avogadro/mainwindow.cpp

```diff
@@ -1496,7 +1502,7 @@ void MainWindow::buildMenu()
 
 #ifdef Q_OS_MAC
   // hide the file toolbar on Mac
-  m_fileToolBar->hide();
+//  m_fileToolBar->hide();
```

Comment:
Why hide it if we will never show it on the Mac anyway? I never saw this in my testing, but it seems a reasonable change unless I am missing a good reason we would want to go back. I would be in favor of just removing it, it will still be in the file history.